### ISSUE TITLE
classList not set correctly on Webcam video element

### ISSUE THREAD

**Issue by thuna-garry** [AlexKratky/vue-camera-lib#2]
Wonderful and clean utility.  Many thanks.

The "classList" property on the  Webcam's video element is not correctly setting parent specified classes.  Instead it is adding the literal "classList" as a class.

Instead, I suggest the video element should be changed from 

```
    <video
        ref="video"
        class="w-full h-auto"
        :class="{'hidden': ! deviceId, classList: true}"
        autoplay
    />

```
to
```
    <video
        ref="video"
        :class="[{'hidden': ! deviceId}, ...classList.split(' ')]"
        autoplay
    />

```

As it looks like the WebcamUI component simply uses embeds the Webcam component, this should fix both.

Thanks again.

**Comment by AlexKratky** [comment]
Hello @thuna-garry, thank you for reaching out, I am glad that this utility helped you. Thank you for pointing that out, I will fixed this when I will have more free time.

**Comment by AlexKratky** [comment]
Fixed in 499bd6ca9e8b253bf805bde94405e6ea7d71e99c
